From ab6dd9a8e5f2a80ad50a009830b2ddac6d88086f Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Mon, 19 Feb 2024 10:42:28 +0100
Subject: [PATCH] Use QtWebOSEglClientBuffer for cmake

:Release Notes:

:Detailed Notes:

:Testing Performed:
Only build tested.

:QA Notes:
No change to image.

:Issues Addressed:
[WRP-310] Apply .patch files for hot-fix to corresponding repositories
[WRP-11630] Add Upstream-Status to existing .patch files in meta-lg-webos
[WRP-11864] CCC: Various build fixes and .patch files cleanup

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Submitted [http://gpro.lge.com/c/webosose/qtwayland-webos/+/349070 Use QtWebOSEglClientBuffer for cmake]

 sync.profile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sync.profile b/sync.profile
index 1690cf6..33361eb 100644
--- a/sync.profile
+++ b/sync.profile
@@ -1,3 +1,3 @@
 %modules = ( # path to module name map
-    "WebOSEglClientBuffer" => "$basedir/src/plugins/hardwareintegration/compositor/webos-wayland-egl/webos-wayland-egl-support",
+    "QtWebOSEglClientBuffer" => "$basedir/src/plugins/hardwareintegration/compositor/webos-wayland-egl/webos-wayland-egl-support",
 );
